### PR TITLE
lis-hotfix-release-pipeline: Modification to support test input from parameter

### PR DESCRIPTION
**Pull request by johnsongeorge-w**

### CODE REVIEW

**Comment by johnsongeorge-w** [comment]
@iamshital @prashantmhetar : Review the modifications for supporting the test input as parameter.

**Comment by johnsongeorge-w** [comment]
Changes verified: Job:347 of pipeline-lis-hotfix

File: linux_pipeline/Jenkinsfile_azure_lis_hotfix_performance [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
-    "westus2"],
-    ["Storage4K", "PERF-STORAGE-4K-IO", "westus2"],
-    ["Storage1024K", "PERF-STORAGE-1024K-IO", "northeurope"],
-    ["NVME4K", "PERF-NVME-4K-IO", "westus2"],
-    ["UDPNetworkPerformance_Multiclient", "PERF-NETWORK-UDP-1K-THROUGHPUT-MULTICONNECTION-NTTTCP-Synthetic,PERF-NETWORK-UDP-1K-THROUGHPUT-MULTICONNECTION-NTTTCP-SRIOV", "westus2"],
-    ["PERF-SYSCALL-BENCHMARK", "PERF-SYSCALL-BENCHMARK", "westus2"]
-]
-def PerfTestCount = PerfTestList.size()
+
+def TestLocation = TEST_LOCATION.replaceAll("[\n\r]", "");
```
Comment (iamshital) [inline linux_pipeline/Jenkinsfile_azure_lis_hotfix_performance]: We are taking TEST_LOCATION value as single string. No need to replaceAll()

File: linux_pipeline/Jenkinsfile_azure_lis_hotfix_performance [hunk 2]
```diff
@@ -1,15 +1,10 @@
...
+def PerfTestList = PERFORMANCE_TESTS.replaceAll("[\n\r]", "").split(',');
+def stageTimeout = 90   // Set the timeout for each stage as 90 minutes
```
Comment (iamshital) [inline linux_pipeline/Jenkinsfile_azure_lis_hotfix_performance]: Please remove timeout. If we see longer running tests then we can introduce it in future. We don't want tests to be stopped due to timeout.

File: linux_pipeline/Jenkinsfile_azure_lis_hotfix_performance (hunk 1 above)
Comment (johnsongeorge-w) [reply linux_pipeline/Jenkinsfile_azure_lis_hotfix_performance]: Done

File: linux_pipeline/Jenkinsfile_azure_lis_hotfix_performance (hunk 2 above)
Comment (johnsongeorge-w) [reply linux_pipeline/Jenkinsfile_azure_lis_hotfix_performance]: Done

**Pull request merged**